src/LuaInterface.cpp

```diff
@@ -565,23 +566,23 @@ void LuaInterface::renameVar(TVar* var)
     //this assumes anything like reparenting has been done
     L = interpreter->pGlobalLua;
     QList<TVar*> vars = varOrder(var);
-    QString oldName = vars[0]->getName();
+    QString oldVariableDeleteCode = vars[0]->getName();
```

Comment:
QString oldVariableDeleteCode = vars.at(0)->getName();

I _think_ will compile and `QList<T>::at()` is _better_ here and beyond, than using `QList<T>::operator[]`...